src/app/functions/sendPageviewEvent.js

```diff
@@ -0,0 +1,31 @@
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * You must provide ReactRouterDOM location object (or equivalent) from a React function component where you can useLocation.
+ * 
+ * @param {object} location - ReactRouterDOM location object 
+ */
+const sendPageviewEvent = (location, title = false) => {
+    
+    let payload = {
+        category: 'Admin', // to mirror Endurance\WP\Module\Data\Listeners\Admin()
+        action: 'pageview',
+        data: {}
+    };
+
+    payload.data.page           = window.bluehostWpAdminUrl + '?page=bluehost#' + location.pathname; // full url
+    if ( title ) {
+        payload.data['page_title']  = title;
+    }
+    if ( ('undefined' !== typeof location.state) && ('undefined' !== typeof location.state.redirect ) ) {
+        payload.data.redirect = location.state.redirect;
+    }
```

Comment:
I left [a comment on the other PR for the hub side](https://github.com/bluehost/bluehost-wordpress-hub/pull/20#issuecomment-754705182), but I'm wondering if we should send these redirects using a different `action` so they fire as events instead.

Reply:
@earnjam I think we'd need to fire two separate hits, as the redirect payload is effectively attached to a pageview hit we want to record and it's not really an "action" or true event, that's just where I was trying to jam the data for simplicity. I think these are best delivered as Custom Dimensions on the pageview hits.